data.sql

```diff
@@ -22,3 +22,38 @@ UPDATE animals SET owner_id = 2 WHERE name IN ('Gabumon', 'Pikachu');
 UPDATE animals SET owner_id = 3 WHERE name IN ('Devimon', 'Plantmon');
 UPDATE animals SET owner_id = 4 WHERE name IN ('Charmander', 'Squirtle', 'Blossom');
 UPDATE animals SET owner_id = 5 WHERE name IN ('Angemon', 'Boarmon');
+
+INSERT INTO vets (name, age, date_of_graduation) VALUES
+('William Tatcher', 45, '2000-04-23'),
+('Maisy Smith', 26, '2019-01-17'),
+('Stephanie Mendez', 64, '1981-05-04'),
+('Jack Harkness', 38, '2008-06-08');
+
+INSERT INTO specializations (vet_id, species_id)
+VALUES
+  (1, 1),
+  (3, 1),
+  (3, 2),
+  (4, 2);
+
+INSERT INTO visits (animal_id, vet_id, visit_date) VALUES
+(1, 1, '2020-05-24'),
+(1, 3, '2020-07-22'),
+(2, 4, '2021-02-02'),
+(3, 2, '2020-01-05'),
+(3, 2, '2020-03-08'),
+(3, 2, '2020-05-14'),
+(4, 3, '2021-05-04'),
+(6, 4, '2021-02-24'),
+(7, 2, '2019-12-21'),
+(7, 1, '2020-08-10'),
+(7, 2, '2021-04-07'),
+(8, 3, '2019-09-29'),
+(9, 4, '2020-10-03'),
+(9, 4, '2020-11-04'),
+(10, 2, '2019-01-24'),
+(10, 2, '2019-05-15'),
+(10, 2, '2020-02-27'),
+(10, 2, '2020-08-03'),
+(11, 3, '2020-05-24'),
+(11, 1, '2021-01-11');
```

Comment:
## Your Animals Table
![image](https://github.com/edgardsoza/vet-clinic-database/assets/93039359/719d2376-bbeb-41d4-b783-a02a7184f4b5)

- [ ] It would be nice if you look at your animal id because I don't think that you have any animal with `id = 11` Actually, the problem starts at line 47 there should be `5` instead of `6` and so on all the next insertions messed up from line 47 to 59 😨 Kindly fix them to make your project follow the requirements 👍